### ISSUE TITLE
xdai bootnodes list is incorrect

### ISSUE THREAD

**Issue by jkilpatr** [poanetwork/poa-chain-spec#109]
```
Starting Nmap 7.60 ( https://nmap.org ) at 2019-03-06 13:31 EST
Nmap scan report for 40.87.15.152
Host is up (0.025s latency).
Not shown: 998 filtered ports
PORT    STATE SERVICE  VERSION
22/tcp  open  ssh      OpenSSH 7.2p2 Ubuntu 4ubuntu2.8 (Ubuntu Linux; protocol 2.0)
443/tcp open  ssl/http nginx 1.10.3 (Ubuntu)
Service Info: OS: Linux; CPE: cpe:/o:linux:linux_kernel

Service detection performed. Please report any incorrect results at https://nmap.org/submit/ .
Nmap done: 1 IP address (1 host up) scanned in 17.57 seconds
[justin@DESKTOP-UALBV95 ~]$ nmap -sV 13.66.219.225

Starting Nmap 7.60 ( https://nmap.org ) at 2019-03-06 13:32 EST
Nmap scan report for 13.66.219.225
Host is up (0.074s latency).
Not shown: 998 filtered ports
PORT    STATE SERVICE  VERSION
22/tcp  open  ssh      OpenSSH 7.2p2 Ubuntu 4ubuntu2.8 (Ubuntu Linux; protocol 2.0)
443/tcp open  ssl/http nginx 1.10.3 (Ubuntu)
Service Info: OS: Linux; CPE: cpe:/o:linux:linux_kernel
```

The listed servers do not have Parity available on port 30303

tl;dr There's no public way to run your own xdai node, you must use the official RPC or know someone who knows the public address of one of the validator nodes

**Comment by phahulin** [comment]
Discovery protocol uses UDP. Please make sure you don't have any restrictions/firewall settings that may block udp traffic. I can successfully sync blockchain with those two bootnodes.

**Comment by sbc64** [comment]
So we were running the wrong spec. Thanks for the help.
